packages/@aws-cdk/applet-js/package.json

```diff
@@ -25,16 +25,9 @@
   "license": "Apache-2.0",
   "devDependencies": {
     "@types/fs-extra": "^5.0.5",
-    "@types/yaml": "^1.0.2",
```

Comment:
Throw in a `"deprecated": "<message>"` in there?

Reply:
Done.